src/metabase/query_processor/util/nest_query.clj

```diff
@@ -50,9 +50,48 @@
                       all-fields)))
     outer-query))
 
+(defn- remove-namespaced-options [options]
+  (when options
+    (not-empty (into {}
+                     (remove (fn [[k _]]
+                               (qualified-keyword? k)))
+                     options))))
```

Comment:
Not certain if we want to reuse it, but do we actually want [disqualify](https://github.com/metabase/metabase/blob/master/src/metabase/lib/convert.cljc#L398)?

That is, a more specific removal of certain namespaces, rather than all of them? Perhaps its not worth worrying about, with this namespace due to be overhauled.